benchmark/e2e/README.md

```diff
@@ -0,0 +1,60 @@
+# End-to-End Benchmarking
+
+The purpose of this folder is to be able to test the overhead of dd-trace on an
+application. The primary focus here is on easily measurable metrics like
+latency, RPS and throughput.
+
+We're using a sample app called AcmeAir, which is used by Node.js to benchmark
+itself (results are at <https://benchmarking.nodejs.org/>). Load is produced
+with [autocannon](https://npm.im/autocannon), which also gives us results. We
+test with and without the tracer to get a measure of overhead. We test using two
+separate endpoints that are measured independently, to get a measure of worst
+case (a static landing page) and a more realistic case (a DB call is done).
+
+## Requirements
+
+This test should work with all versions of Node.js supported by dd-trace. In
+addition, the sample app uses MongoDB, so you'll have to have that running and
+listening on the default port. If you're set up with the `docker-compose.yml` in
+the root of this repo, you should be ready.
+
+## Usage
+
+To start the test, run `node test` in this directory. This will install AcmeAir
+if it hasn't yet been installed, and populate MongoDB if that hasn't already
+been done.
+
+Next, it will run the test for 10 seconds each on each of the 2 endpoints, both
+with and without the tracer loaded. That means 40 seconds of testing. Results
+will appear on stdout.
+
+You can change the duration of the tests by setting the `DURATION` environment
+variable to the number of seconds to run. Keep in mind that this will be run 4
```

Comment:
It's actually 6, since I forgot to update the doc: 1 each of control (no tracer), async_hooks only, and with tracing enabled, all of that across two different endpoints.